src/dashboard/src/components/administration/tests/test_processing_config.py

```diff
@@ -93,3 +95,21 @@ def test_name_field_is_validated(self, mock_conf_fields):
             response,
             "The name can contain only alphanumeric characters and the underscore character (_).",
         )
+
+    def test_reset_default_processing_config(self):
+        response = self.client.get("/administration/processing/reset/default/")
+        self.assertEqual(response.status_code, 302)
+        processing_config = os.path.join(
+            helpers.processing_config_path(), "defaultProcessingMCP.xml"
+        )
+        with open(processing_config) as actual_file:
+            assert actual_file.read() == DEFAULT_PROCESSING_CONFIG
+
+    def test_reset_automated_processing_config(self):
+        response = self.client.get("/administration/processing/reset/automated/")
+        self.assertEqual(response.status_code, 302)
+        processing_config = os.path.join(
+            helpers.processing_config_path(), "automatedProcessingMCP.xml"
+        )
+        with open(processing_config) as actual_file:
+            assert actual_file.read() == AUTOMATED_PROCESSING_CONFIG
```

Comment:
I'm realizing these tests will likely fail unless the environment has created the shared directory beforehand. Maybe this is true in many other places, I haven't checked. Not entirely sure what would be a good way to get around that, maybe patching django_settings.SHARED_DIRECTORY with the path of a temporary directory created by this test?

Reply:
FYI I'm fixing this from a different pull request. It passed with `--randomly-seed=1618426567` because test_api is running first.